### DISCUSSION TITLE
How can I obtain the results from the testing process?

### DISCUSSION THREAD

**Discussion by xiaoqi-neu1** [deepmodeling/deepmd-kit#3091] [Q&A]
For the water molecule dataset, after testing, I want to plot regression lines for both energy and force. Currently, the screen only displays the MAE result. How can I obtain all the results from the testing process to plot the regression line?

dp train input.json
dp freeze -o graph.pb
dp compress -i graph.pb -o graph-compress.pb
dp test -m graph-compress.pb -s ../data/data_valid -n 40 -d results

DEEPMD INFO    # number of test data : 1015 
DEEPMD INFO    Energy MAE         : 1.674906e+02 eV
DEEPMD INFO    Energy RMSE        : 6.160213e+02 eV
DEEPMD INFO    Energy MAE/Natoms  : 2.584731e-01 eV
DEEPMD INFO    Energy RMSE/Natoms : 9.506501e-01 eV
DEEPMD INFO    Force  MAE         : 1.497496e-02 eV/A
DEEPMD INFO    Force  RMSE        : 2.666862e-02 eV/A
DEEPMD INFO    Virial MAE         : 3.588804e+00 eV
DEEPMD INFO    Virial RMSE        : 6.192421e+00 eV
DEEPMD INFO    Virial MAE/Natoms  : 5.538278e-03 eV
DEEPMD INFO    Virial RMSE/Natoms : 9.556206e-03 eV
DEEPMD INFO    # -----------------------------------------------

**Comment by wanghan-iapcm** [comment]
should be in files result.*.out
